### PR TITLE
New Prompt Component

### PR DESCRIPTION
**Pull request by Juveniel**
targets: https://github.com/telerik/kendo-themes-private/issues/175

### CODE REVIEW

File: packages/bootstrap/scss/prompt/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
+// Prompt
+
+/// The text color of the of the Prompt.
```
Comment (epetrow) [inline packages/bootstrap/scss/prompt/_variables.scss:3]: There are some redundant words in the variable description that should be omitted: 

The text color of the <ins>of the</ins>, The background color of the <ins>of the</ins>, The border color of the <ins>of the</ins>

_Note:_ This appears from line 3 through 26

File: packages/classic/scss/prompt/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
+// Prompt
+
+/// The text color of the of the Prompt.
```
Comment (epetrow) [inline packages/classic/scss/prompt/_variables.scss:3]: Same here, redundant use of:  <ins>of the</ins> lines 3 through 26

File: packages/default/scss/prompt/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
+// Prompt
+
+/// The text color of the of the Prompt.
```
Comment (epetrow) [inline packages/default/scss/prompt/_variables.scss:3]: Here as well, line 3 through 26 -> remove extra <ins>of the</ins>

File: packages/fluent/scss/prompt/_layout.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        flex: 0 0 auto;
+        gap: var( --kendo-prompt-expander-spacing, #{$kendo-prompt-expander-spacing} );
+        align-self: stretch;
+    }
+
+    .k-prompt-suggestion {
+        border-width: 1px;
+        border-style: solid;
+        border-radius: var( --kendo-prompt-suggestion-border-radius, #{$kendo-prompt-suggestion-border-radius} );
+        padding-block: var( --kendo-prompt-suggestion-padding-x, #{$kendo-prompt-suggestion-padding-y} );
```
Comment (epetrow) [inline packages/fluent/scss/prompt/_layout.scss:46]: Shouldn't it be `var( --kendo-prompt-suggestion-padding-y, ...` here?

File: packages/fluent/scss/prompt/_theme.scss [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    .k-prompt {
+        @include fill(
+            var( --kendo-prompt-text, #{$kendo-prompt-text} ),
+            var( --kendo-prompt-bg, #{$kendo-prompt-bg} ),
+            var( --kendo-prompt-border, #{$kendo-prompt-border} )
+        );
+    }
+
+    .k-prompt-header {
+        @include fill(
```
Comment (epetrow) [inline packages/fluent/scss/prompt/_theme.scss:15]: Shouldn't we use `$kendo-prompt-header-*` here?

File: packages/fluent/scss/prompt/_variables.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
+@use "sass:map";
+@use "../core/" as *;
+
+/// The text color of the of the Prompt.
```
Comment (epetrow) [inline packages/fluent/scss/prompt/_variables.scss:4]: Here also, line 4 through 27 -> remove redundant <ins>of the</ins>

File: packages/material/scss/prompt/_variables.scss [hunk 7]
```diff
@@ -0,0 +1,66 @@
+// Prompt
+
+/// The text color of the of the Prompt.
```
Comment (epetrow) [inline packages/material/scss/prompt/_variables.scss:3]: Same here, line 3 through 26 -> remove unnecessary <ins>of the</ins>

**Review by epetrow — CHANGES_REQUESTED** [review]

**Comment by epetrow** [comment]
Additionally in the design the prompt actions are aligned with the TextArea which is no the case in the visual tests (there is a difference in the block padding)

**Comment by Juveniel** [comment]
> Additionally in the design the prompt actions are aligned with the TextArea which is no the case in the visual tests (there is a difference in the block padding)

Discussed this with @dmanova and the issue turns out to be design related. We want the content area be spaced inwards, whereas the toolbar and action buttons are aligned.

**Review by epetrow — APPROVED** [review]
lgtm

**Pull request merged**